### ISSUE TITLE
nf-seti-go2 not rendering

### ISSUE THREAD

**Issue by Satont** [ryanoasis/nerd-fonts#1032]
#### 🗹 Requirements

- [x] I have searched the [issues](https://github.com/ryanoasis/nerd-fonts/issues) for my issue and found nothing related and/or helpful
- [x] I have searched the [FAQ](https://github.com/ryanoasis/nerd-fonts/wiki/FAQ-and-Troubleshooting) for help
- [x] I have searched the [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki) for help

#### 🎯 Subject of the issue

Experienced behavior:
No icon (other icons work, atleast most of them)
![image](https://user-images.githubusercontent.com/42675886/209757862-8e307a97-1fc6-4768-bc69-389c98446ea5.png)

Expected behavior:
Should be rendered
Example symbols:

nf-seti-go2
#### 🔧 Your Setup

- _Which font are you using (e.g. `Anonymice Powerline Nerd Font Complete.ttf`)?_
  `nerd-fonts-jetbrains-mono`
- _Where did you get the file from (download link, self patched, source downloaded from link...)_
   - arch repos
- _Which terminal emulator are you using (e.g. `iterm2`, `urxvt`, `gnome`, `konsole`)?_
   - alacritty
- _Are you using OS X, Linux or Windows? And which specific version or distribution?_
   - Linux

**Comment by Finii** [comment]
You are right, the cheat sheet shows all icons up to the latest `v2.3.0` HEAD.

In comparison to `v2.2.2` a lot of icons are (will be) added to `v2.3.0`. When I worked on the website two (?) weeks ago and fixes some issues there I also updated the cheat sheet (already) to what `v2.3.0` will be.

Compare
* #978 
* 974249b5 `Actually use new Seti`
* 4fba05a7 `CI: Keep i_seti.sh in sync with autogenerated font`
* 3de381d5a `CI: Automatically deploy Cheat Sheet`
* 7fe81d25 `Update Cheat Sheet`

If you need I can create the Jet Brains font set with up to date icons. I intended to release `v2.3.0` just before Christmas, something that obviously failed :-(

**Comment by Satont** [comment]
Cool. I'll wait for release, no rush.

**Comment by Finii** [comment]
Released. Should be there :-)
